packages/mdc-tab/mdc-tab.scss

```diff
@@ -69,22 +70,21 @@
 }
 
 .mdc-tab__text-label {
-  @include mdc-typography-base;
-
   display: inline-block;
   padding: 16px 0 20px;
-  font-size: 14px;
-  font-weight: 500;
+  opacity: $mdc-tab-text-label-opacity;
+  line-height: 1;
```

Comment:
Is this overriding `mdc-typography(button)` on `.mdc-tab` above? If so, should we just put this there with a comment to make it clearer?

Reply:
Sounds good